src/formpack/schema/fields.py

```diff
@@ -139,7 +158,7 @@ def _get_label(
         # even if `lang` can be None, we don't want the `label` to be None.
         label = self.labels.get(lang, self.name)
         # If `label` is None, no matches are found, so return `field` name.
-        return self.name if label is None else label
+        return label or self.name
```

Comment:
👀 , even If I prefer your change, it does not do the same thing. if `label` is empty (not None), `self.name` would be returned. Is it really what we want? I guess so.

Reply:
The change was intentional as I don't think the previous logic was correct. AFAIR I was causing some of my new tests to fail so I changed it 👌

Reply:
I somewhat doubt that it's related, but we need careful handling of labels and more unit test coverage. I've found a regression related to choice names with missing translations: https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/formpack.20and.20missing.20labels/near/173165. I'll make a correction or open a new issue as needed.

Reply:
#307